great_expectations/render/types/__init__.py

```diff
@@ -501,44 +508,91 @@ class RenderedAtomicValue(DictDot):
     def __init__(
         self,
         schema: Optional[dict] = None,
+        header: Optional["RenderedAtomicValue"] = None,
         template: Optional[str] = None,
         params: Optional[dict] = None,
-        header: Optional["RenderedAtomicValue"] = None,
         header_row: Optional[List["RenderedAtomicValue"]] = None,
         table: Optional[List[List["RenderedAtomicValue"]]] = None,
         graph: Optional[dict] = None,
         kwargs: Optional[dict] = None,
     ) -> None:
         self.schema: Optional[dict] = schema
+        self.header: Optional[RenderedAtomicValue] = header
 
         # StringValueType
         self.template: Optional[str] = template
         self.params: Optional[dict] = params
 
         # TableType
-        self.header: Optional[RenderedAtomicValue] = header
         self.header_row: Optional[List[RenderedAtomicValue]] = header_row
         self.table: Optional[List[List[RenderedAtomicValue]]] = table
 
         # GraphType
-        self.graph: Optional[RenderedAtomicValue] = graph
+        self.graph: Optional[RenderedAtomicValueGraph] = RenderedAtomicValueGraph(
+            graph=graph
+        )
 
         # UnknownType
         self.kwargs: Optional[dict] = kwargs
 
+    def __repr__(self) -> str:
+        return json.dumps(self.to_json_dict(), indent=2)
+
+    def __str__(self) -> str:
+        return json.dumps(self.to_json_dict(), indent=2)
+
+    def to_json_dict(self) -> dict:
+        """Returns RenderedAtomicValue as a json dictionary."""
+        d = renderedAtomicValueSchema.dump(self)
+        if "schema" in d:
+            d["schema"] = self.schema
+        if "header" in d:
+            d["header"] = self.header
+        if "template" in d:
+            d["template"] = self.template
+        if "params" in d:
+            d["params"] = self.params
+        if "header_row" in d:
+            d["header_row"] = self.header_row
+        if "table" in d:
+            d["table"] = self.table
+        if "graph" in d:
+            d["graph"] = self.graph.to_json_dict()
+        if "kwargs" in d:
+            d["kwargs"] = self.kwargs
```

Comment:
I get the need for this but is there a way to make it a bit more concise/maintainable? Can't think of it since `self.graph` is an edge case but just thinking out loud.

Reply:
I was able to refactor this with `getattr` ✅